### DISCUSSION TITLE
Adding the option to use properties instead of tags for creating decks

### DISCUSSION THREAD

**Discussion by lightningRalf** [st3v3nmw/obsidian-spaced-repetition#959] [General]
I would like to use tags really specific. And with wanting to use a lot of different decks. I think the tags will get crowded really fast.
So my question is, how hard would it be to use properties (frontmatter) to do this instead?
My thinking are for two options:
1. specify a general property: / flashcard:: {nameOfDeck}
or
2. {nameOfDeck}:: {true}
with a setting (just like it is currently for tags) where you put the nameOfDeck inside a box 

Thanks in advance for considering.
Greetings from Franconia - Germany.

**Comment by ronzulu** [comment]
Hi @lightningRalf greetings from Sydney

I'm curious what problem you are trying to solve.

The plugin already supports having lots of decks (organized in a hierarchy of decks). e.g. `#flashcards/computing/cloud`

Cheers
Ronny

**Comment by lightningRalf** [comment]
I am trying to avoid cluttering the tags.
But I think the nested tags version should suffice. Better time spend elsewhere.
